### PR TITLE
Add support for non-virtual fairs

### PR DESCRIPTION
**Pull request by julianweng**
Conditionally removes copy related specifically to virtual club fairs to support the usage of Penn Clubs fairs for solely informational purposes without using the remaining (mostly virtual) functionality.

Creates default help page for club fairs that uses the provided descriptions instead of hardcoded information.

We should also add the option to toggle between external registration and Clubs-managed registration (opt out of the latter) for the fairs at some point but these changes are higher priority given the incorrect banner on the homepage.

### CODE REVIEW

File: frontend/components/ClubPage/LiveEventsDialog.tsx [hunk 1]
```diff
@@ -121,8 +123,8 @@ const LiveEventsDialog = ({
       </Link>
       <LiveTitle>{fairName}</LiveTitle>
       <LiveSub>
-        {liveEventCount === 0 ? (
-          `Get ready for the virtual ${FAIR_NAME} fair! If you need help registering email contact@pennclubs.com`
```
Comment (aviupadhyayula) [inline frontend/components/ClubPage/LiveEventsDialog.tsx:125]: Why are we taking out the contact email? Can we point them to OSA instead?

File: frontend/pages/fair.tsx [hunk 2]
```diff
@@ -78,29 +74,43 @@ const FairPage = ({
...
-          College Green from 12p-4p each day.
-        </p>
-        <p>
-          To participate, sign-up will be facilitated through Penn Clubs and
-          will coincide with the annual club registration process. Only
-          returning undergraduate student-run groups that were registered in
-          Penn Clubs last year are eligible to sign-up for the Fall Activities
-          Fair.
+          The {fair?.name} sponsored by {fair?.organization}, will be held from{' '}
+          {fair?.start_time.split('T')[0]} to {fair?.end_time.split('T')[0]}!
```
Comment (aviupadhyayula) [inline frontend/pages/fair.tsx:78]: Nit: can we throw in defaults here, in case `fair` is null?

**Review by aviupadhyayula — COMMENTED** [review]
Thanks for picking this up Julian, LGTM. If you have time, it'd be nice to throw in a quick unit test for creating a virtual fair.

**Review by rm03 — APPROVED** [review]
Looks good to me, thanks Julian

**Comment by julianweng** [comment]
Thanks for the review, I added the private option to the existing fair creation test, but there isn't any difference in backend behavior caused by it, so would like to condition a separate unit test on that being the case.

**Pull request merged**